neuromation/api/quota.py

```diff
@@ -8,7 +8,6 @@
 
 @dataclass(frozen=True)
 class _QuotaInfo:
-    name: str
```

Comment:
we could still store a cluster name here.

Reply:
Why we need name here?
Dict key will be more useful for end-user

Reply:
`QuotaInfoFormatter` could unambiguously output the cluster name for example.

Reply:
added